protobufs/livekit_analytics.proto

```diff
@@ -202,3 +204,24 @@ message AnalyticsNodeRooms {
   google.protobuf.Timestamp timestamp = 3;
   repeated AnalyticsRoom rooms = 4;
 }
+
+message ReportInfo {
```

Comment:
Container for `Report` event data. Starting off a `FeatureUsage` to report Krisp usage.